### DISCUSSION TITLE
If I close the gui, will the full node keep running? (Linux)

### DISCUSSION THREAD

**Discussion by dbolser** [Chia-Network/chia-blockchain#5398] [General]
I'm running this over ssh. Can I just close the gui and expect it to keep running?

kthxbi.

**Comment by climba03003** [comment]
I think no, the gui will close all the chia related programs even if it is not started by itself.

**Reply by dbolser** [reply]
Thanks @climba03003, I've seen plenty of tutorials, but they are all too in-depth... how do I simply 'start chia' from the cli?

Many thanks again.

**Reply by climba03003** [reply]
`chia start all` / `chia start farmer`
Either one can be used to farm

**Reply by lvcivs** [reply]
Use `chia start farmer` instead of `chia start all`, the latter will try to start a timelord which is not recommended for normal users.

**Reply by dbolser** [reply]
Thanks both. I'm not quite a normal user... How do I know if being a timelord is for me?

**Reply by dbolser** [reply]
So after closing the gui, logging off and then back on again, hitting `. ./activate` and then `chia start farmer` I see the following:
```
chia_harvester: Already running, use `-r` to restart
chia_farmer: Already running, use `-r` to restart
chia_full_node: Already running, use `-r` to restart
chia_wallet: Already running, use `-r` to restart
```

Safer to restart? How to check 'status'?

Huge thanks again both.

**Reply by dbolser** [reply]
OK, `chia show -s` gives me:
```
Current Blockchain Status: Full Node syncing to block 299449 
Currently synced to block: 3175
Current Blockchain Status: Not Synced. Peak height: 3175
...
```
